pt-BR/mail/chrome/messenger-smime/msgSecurityInfo.properties

```diff
@@ -37,33 +37,33 @@ EIInvalidLabel=A mensagem não pode ser descriptografada.
 EIInvalidHeader=Esta mensagem foi criptografada antes de ser enviada a você, mas não pode ser descriptografada.
 
 EIContentAltered=O conteúdo da mensagem aparenta ter sido modificado durante a transmissão.
-EINoDecryptCert=The certificate used to encrypt the message cannot be found.
-EIPasswordError=You did not enter your Master Password correctly.
-EIInvalidCipher=The message was encrypted using an encryption strength that this version of your software does not support.
+EINoDecryptCert=O certificado usado para criptografar a mensagem não pode ser encontrado.
+EIPasswordError=Você não digitou a senha mestra corretamente.
+EIInvalidCipher=A mensagem foi criptografada usando um tipo de criptografia que o seu aplicativo não suporta.
```

Comment:
CORRECTIONS NEEDED it should be EIInvalidCipher=A mensagem foi criptografada usando um tipo de criptografia que esta versão do seu aplicativo não suporta.